docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -142,24 +147,24 @@ Lightly Platform:
 
 .. image:: ./images/webapp-explore-after-docker.jpg
 
-In our case, we had 4 short street videos with about 1000 frames each in the S3 
+In our case, we had 4 short street videos with about 1000 frames each in the 
 bucket and selected 50 frames from it. Now you can analyze your dataset in the
 embedding and metadata view of the Lightly Platform, subsample it further, or 
 export it for labeling. In our case we come to the conclusion that the raw data 
 we have does not cover enough cases and thus decide that we want to first 
 collect more street videos.
 
-Process new data in your S3 bucket using a datapool
-------------------------------------------------------
-You probably get new raw data from time to time added to your S3 bucket. In our 
-case we added 4 more street videos to the S3 bucket. The new raw data might 
+Process new data in your bucket using a datapool
+------------------------------------------------
+You probably get new raw data from time to time added to your bucket. In our 
```

Comment:
Suggestion;
You probably get new raw data from time to time added to your bucket. -> Over time you most likely will be receiving new raw data from your various sources and add these to your cloud storage bucket.